### ISSUE TITLE
Add grouping in Dev wiki for companion-computer topics

### ISSUE THREAD

**Issue by hamishwillee** [diydrones/ardupilot-wiki-issue-tracker#52]
These are all related to companion computers - a growing field

http://dev.ardupilot.com/wiki/raspberry-pi-via-mavlink/
http://dev.ardupilot.com/wiki/odroid-via-mavlink/
http://dev.ardupilot.com/wiki/edison-for-drones/

It may be good to create a main entry on the TOC about using companion computers, and then link these from there. 

Alternatively, if we move to a categorisation/tagged approach, a category.

**Comment by gmorph** [comment]
Yep a TOC landing page about using companion computers sounds great.

Thanks

Grant

On 19 February 2015 at 13:01, Hamish Willee <notifications@github.com>
wrote:

> These are all related to companion computers - a growing field
>
> http://dev.ardupilot.com/wiki/raspberry-pi-via-mavlink/
> http://dev.ardupilot.com/wiki/odroid-via-mavlink/
> http://dev.ardupilot.com/wiki/edison-for-drones/
>
> It may be good to create a main entry on the TOC about using companion
> computers, and then link these from there.
>
> Alternatively, if we move to a categorisation/tagged approach, a category.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot-wiki-issue-tracker/issues/52>.
>



-- 
Thanks

Grant

**Comment by hamishwillee** [comment]
How about this one: http://dev.ardupilot.com/wiki/companion-computers/

Anything else we need to say? Is this the right place on the menu?

**Comment by gmorph** [comment]
I updated it - hope that was ok.

Yep, I reckon that's the right place in the menu's.

Thanks, Grant.

On 24 February 2015 at 21:35, Hamish Willee <notifications@github.com>
wrote:

> How about this one: http://dev.ardupilot.com/wiki/companion-computers/
>
> Anything else we need to say? Is this the right place on the menu?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot-wiki-issue-tracker/issues/52#issuecomment-75733563>
> .
>



-- 
Thanks

Grant

**Comment by hamishwillee** [comment]
Hi Grant,

Thanks for the update - making it clear what is returned to the computer and what it can do with the information is a very important point. I played with the English a bit and removed the bit where I specified "pixhawk" (since the concept of a companion computer could theoretically be used with any controller, even though we only cover one).

I'm closing this as "done". Thanks for your help.

Regards
H
